charts/cryostat/templates/storage_secret.yaml

```diff
@@ -1,7 +1,9 @@
+{{- if empty .Values.storage.storageSecretName -}}
 apiVersion: v1
 kind: Secret
 metadata:
-  name: {{ .Release.Name }}-storage
+  name: {{ .Release.Name }}-storage-secret
```

Comment:
I think Secrets should have the common labels, similar to other resource. This applies to all generated secrets.

```
  labels:
    {{- include "cryostat.labels" $ | nindent 4 }}
```